rust/examples-frontend/echo/src/index.ts

```diff
@@ -10,10 +10,10 @@ type ConsumerId = Brand<string, 'ConsumerId'>;
 type ProducerId = Brand<string, 'ProducerId'>;
 
 interface ServerInit {
-    action: 'Init';
-    consumerTransportOptions: TransportOptions;
-    producerTransportOptions: TransportOptions;
-    routerRtpCapabilities: RtpCapabilities;
+	action: 'Init';
+	consumerTransportOptions: TransportOptions;
+	producerTransportOptions: TransportOptions;
+	routerRtpCapabilities: RtpCapabilities;
```

Comment:
Why these formatting changes? Please keep current code style untouched.

Reply:
it was made by IDE :c

it doesn't even show that master and my branch differs